### PR TITLE
add lxml

### LINKED ISSUE #117: llvm-mt retuns 'error: no libxml2'
libxml2 appears to be installed on my local computer.

```
miniconda3\Library\bin\llvm-mt.exe /nologo /manifest CMakeFiles\CMakeTmp\CMakeFiles\cmTC_17307.dir/intermediate.manifest /out:CMakeFiles\cmTC_17307.dir/embed.manifest /notify_update
llvm-mt: ignoring unsupported 'notify_update' option
llvm-mt: error: no libxml2
```

### PR DESCRIPTION
**Pull request by xoviat**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

closes #117

### CODE REVIEW

**Comment by xoviat** [comment]
@conda-forge-admin rerneder

**Comment by xoviat** [comment]
@conda-forge-admin rerender

**Comment by xoviat** [comment]
@isuruf I found no evidence in the log that lxml is enabled. Is that a problem?

**Comment by xoviat** [comment]
@isuruf is it possible to revert a97ed9c?

**Comment by isuruf** [comment]
Sure

**Comment by xoviat** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,6 +35,7 @@ requirements:
   host:
     - libcxx {{ cxx_compiler_version }}  # [osx]
     - zlib
+    - libxml2
```
Comment (isuruf) [inline recipe/meta.yaml:38]:
```suggestion
    - libxml2   # [win]
```

**Comment by xoviat** [comment]
@conda-forge-admin rerender

**Comment by xoviat** [comment]
@isuruf Is there anything else? What about this?

> I found no evidence in the log that lxml is enabled. Is that a problem?

**Pull request merged**